tests/docproc/schemamapping/schemamapping.rb

```diff
@@ -14,10 +14,10 @@ def setup
   end
 
   def test_schemamapping_basic
-    feed_and_wait_for_docs("simple", 2, :file => selfdir+"simple.xml", :cluster => "simple", :route => "\"default/chain.apple default/chain.banana indexing\"")
```

Comment:
I guess teh xml files should be removed too.

Reply:
Yes, thanks, fixed